css/styles.css

```
@@ -0,0 +1,264 @@
+@font-face {
+  font-family: Raleway;
+  font-style: normal;
+  font-weight: 800;
+  src: url("../fonts/raleway/raleway-800.woff2") format("woff2");
+  font-display: swap;
+}
+@font-face {
+  font-family: Rubik;
+  font-style: normal;
+  font-weight: 400;
+  src: url("../fonts/rubik/rubik-400.woff2") format("woff2");
+  font-display: swap;
+}
+@font-face {
+  font-family: Rubik;
+  font-style: normal;
+  font-weight: 700;
+  src: url("../fonts/rubik/rubik-700.woff2") format("woff2");
+  font-display: swap;
+}
+@font-face {
+  font-family: Rubik;
+  font-style: normal;
+  font-weight: 800;
+  src: url("../fonts/rubik/rubik-800.woff2") format("woff2");
+  font-display: swap;
+}
+
+/* Стили для html.html */
+
+body {
+  font-family: Rubik, sans-serif;
+  font-size: 18px;
+  line-height: 30px;
+  color: #444444;
+  background-color: #ffffff;
+}
+.page-header,
+.user-navigation-link,
+.main-navigation-link {
+  color: #000000;
+  background-color: #ffe17f;
+  text-decoration: none;
+}
+.user-navigation-list,
+.main-navigation-list,
+.category-product-list,
+.services-list,
+.contact-schedule-list,
+.footer-list,
+.breadcrumbs-list,
+.catalog-filter-list,
+.pagination-list {
+  list-style: none;
+}
+.user-navigation-item:first-child {
+  opacity: 0.3;
+}
+.main-navigation-link,
+.footer-navigation-link {
+  font-family: Raleway, sans-serif;
```

тут лучше писать так, чтобы не дублировать фонт
```suggestion
  font-family: inherit;
```